src/components/frame/Extensions/pages/AdvancedPage/AdvancedPage.tsx

```diff
@@ -4,81 +4,72 @@ import { FormattedMessage, useIntl } from 'react-intl';
-    >
-      <TwoColumns aside={<Navigation pageName="extensions" />}>
-        <h3 className="heading-4 mb-6">
-          {formatMessage({ id: 'advancedPage.colony.title' })}
-        </h3>
-        <div className="mb-4">
-          <NotificationBanner
-            status={canUpgrade ? 'warning' : 'success'}
-            title={
-              <FormattedMessage
-                id={
-                  canUpgrade
-                    ? 'advancedPage.version.warning'
-                    : 'advancedPage.version.success'
-                }
-              />
-            }
-          />
-        </div>
-        <ColonyVersionWidget
-          currentVersion={colonyContractVersion}
-          lastVersion={version || 0}
-          status={canUpgrade ? 'error' : 'success'}
+    <div>
+      <h3 className="heading-4 mb-6">
+        {formatMessage({ id: 'advancedPage.colony.title' })}
+      </h3>
+      <div className="mb-4">
+        <NotificationBanner
+          status={canUpgrade ? 'warning' : 'success'}
+          title={
+            <FormattedMessage
+              id={
+                canUpgrade
+                  ? 'advancedPage.version.warning'
+                  : 'advancedPage.version.success'
+              }
+            />
+          }
         />
-        <div className="divider my-6" />
-        <h3 className="heading-4 mb-6">
-          {formatMessage({ id: 'advancedPage.recovery.title' })}
-        </h3>
-        <p className="text-md text-gray-600 mb-6">
-          {formatMessage({ id: 'advancedPage.recovery.description' })}
-        </p>
-        <div className="mb-6">
-          <NotificationBanner
-            status="info"
-            title={<FormattedMessage id="advancedPage.recovery.notification" />}
-            action={{
-              type: 'redirect',
-              href: 'https://colony.io/colonyjs/docs/colonyjs-core/#recovery-mode',
-              actionText: <FormattedMessage id="text.learnMore" />,
-            }}
-            isAlt
-          />
-        </div>
-        <div className="flex justify-end">
-          {/* @TODO: Add recovery mode logic */}
-          <Button
-            mode="primarySolid"
-            text={{ id: 'button.enterRecoveryMode' }}
-            isFullSize={isMobile}
-          />
-        </div>
-      </TwoColumns>
-    </Spinner>
+      </div>
+      <ColonyVersionWidget
+        currentVersion={colonyContractVersion}
+        lastVersion={version || 0}
+        status={canUpgrade ? 'error' : 'success'}
+      />
+      <div className="divider my-6" />
+      <h3 className="heading-4 mb-6">
+        {formatMessage({ id: 'advancedPage.recovery.title' })}
+      </h3>
+      <p className="text-md text-gray-600 mb-6">
+        {formatMessage({ id: 'advancedPage.recovery.description' })}
+      </p>
+      <div className="mb-6">
+        <NotificationBanner
+          status="info"
+          title={<FormattedMessage id="advancedPage.recovery.notification" />}
```

Comment:
Same question as above, but I see that in a few places we are using `FormattedMessage` and in some not?

Reply:
Just copied it. But will change.